### ISSUE TITLE
change dependency to nest.js <= 9

### ISSUE THREAD

**Issue by jfalch** [wbhob/nest-middlewares#54]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@nest-middlewares/serve-favicon@6.0.1` for the project I'm working on.

we are using nest.js 8, will soon switch to 9, so a dependency to v6 is not really useful, nor necessary: I forced install, and serve-favicon v6.0.1 works well with nest.js v8.4.7.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@nest-middlewares/serve-favicon/package.json b/node_modules/@nest-middlewares/serve-favicon/package.json
index 4ea913c..7b50dd8 100644
--- a/node_modules/@nest-middlewares/serve-favicon/package.json
+++ b/node_modules/@nest-middlewares/serve-favicon/package.json
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/wbhob/nest-middlewares#readme",
   "peerDependencies": {
-    "@nestjs/common": "^6.0.0"
+    "@nestjs/common": "<9.0.0"
   },
   "dependencies": {
     "@types/express": "^4.16.0",
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by davidfmatheson** [comment]
Could we do this for all sub-projects?

**Comment by wbhob** [comment]
just published package version 7.0.0 with compatibility with nestjs 9.0.0
